docs/contributing.md

```diff
@@ -13,11 +13,19 @@ If any notional data is required for the test ensure an appropriately titled fol
 If you would like to run unit tests for the project, you can run   
 ```yarn test ```  
 
-If you would like to run the integration tests for the project please ensure you have selenium server installed and running  
+
+
+If you would like to run the integration tests locally for the project please be sure to first have a running copy of the frontend and ManageIQ backend running locally.  If you don't have them currently running  
```

Comment:
@AllenBW said "mock api", and @chalettu you said MIQ backend.  Can we indicate that either the "mock API or MIQ backend" be running?